### PR TITLE
Fix shortest path crash

### PR DESCRIPTION
**Pull request by nevermore3**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 

close https://github.com/vesoft-inc/nebula-ent/issues/2554
close https://github.com/vesoft-inc/nebula-ent/issues/2558
## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by yixinglu — DISMISSED** [review]
LGTM

File: src/graph/validator/MatchValidator.cpp [hunk 1]
```diff
@@ -165,10 +165,13 @@ Status MatchValidator::buildPathExpr(const MatchPath *path,
...
-    auto min = edgeInfos.front().range->min();
-    if (min != 0 && min != 1) {
-      return Status::SemanticError(
-          "`shortestPath(...)' does not support a minimal length different from 0 or 1");
+    auto *range = edgeInfos.front().range.get();
+    if (range != nullptr) {
+      auto min = range->min();
+      if (min != 0 && min != 1) {
+        return Status::SemanticError(
+            "`shortestPath(...)' does not support a minimal length different from 0 or 1");
```
Comment (xtcyclist) [inline src/graph/validator/MatchValidator.cpp:173]: minor change of this sentence:
`The minimal number of steps for shortestPath() must be either 0 or 1.`

Comment (xtcyclist) [reply src/graph/validator/MatchValidator.cpp:173]: Could try adding a tck to trigger this error.

Comment (nevermore3) [reply src/graph/validator/MatchValidator.cpp:173]: > minor change of this sentence: `The minimal number of steps for shortestPath() must be either 0 or 1.`

refer to the error message corresponding to cypher

Comment (nevermore3) [reply src/graph/validator/MatchValidator.cpp:173]: > 
ok

**Review by yixinglu — APPROVED** [review]

**Review by xtcyclist — APPROVED** [review]

**Pull request merged**